content/W2015/W15-event-teaparty.md

```diff
@@ -0,0 +1,20 @@
+Title: Big CSters Tea Party Social
+Date: 2015-01-25 16:00
+Category: Events
+Tags: social
```

Comment:
I think this should go in a separate commit, but I recommend adding a "big csters" label to this article and the board games one!